### PR TITLE
main/libgphoto2: upgrade to 2.5.22; main/gphoto2: upgrade to 2.5.20

### PR DESCRIPTION
**Pull request by rohfle**

### CODE REVIEW

File: main/gphoto2/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributor: 
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gphoto2
-pkgver=2.5.15
+pkgver=2.5.20
 pkgrel=0
 pkgdesc="Commandline utilities for accessing cameras"
 url="http://www.gphoto.org/"
```
Comment (maxice8) [inline main/gphoto2/APKBUILD]: GPL is not an spdx license

Comment (maxice8) [reply main/gphoto2/APKBUILD]: builddir= declaration can be removed as it is the default value

File: main/libgphoto2/APKBUILD [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libgphoto2
-pkgver=2.5.16
+pkgver=2.5.22
 pkgrel=0
 pkgdesc="The core library of gphoto2, designed to allow access to digital camera by external programs."
 url="http://www.gphoto.org"
```
Comment (maxice8) [inline main/libgphoto2/APKBUILD]: LGPL is not an spdx identifier

Comment (maxice8) [reply main/libgphoto2/APKBUILD]: empty variables can be removed for brevity

Comment (maxice8) [reply main/libgphoto2/APKBUILD]: builddir= declaration can be removed as it is the default value

**Comment by rohfle** [comment]
Thanks - I have updated based on feedback.

**Comment by tcely** [comment]
`master` was upgraded beyond these versions. If the license changes are still needed please rebase this pull request.

**Pull request closed without merging**